velox/expression/Expr.cpp

```diff
@@ -1239,6 +1239,18 @@ bool Expr::applyFunctionWithPeeling(
     newRows = translateToInnerRows(applyRows, *decoded, newRowsHolder);
```

Comment:
should we also be translating finalSelection here?

Reply:
That's a good question. I think the answer is yes. Would you help look into that?

Reply:
sure

Reply:
Thank you. Filed #2595